saleor/extensions/plugins/vatlayer/plugin.py

```diff
@@ -81,30 +81,6 @@ def calculate_checkout_total(
             - checkout.discount
         )
 
-    def calculate_checkout_subtotal(
```

Comment:
Is it expected? Here you are calling this method: https://github.com/mirumee/saleor/pull/4980/files#diff-de9a58978d5e65a9854361c28fb83e80R80

Reply:
Yes. Taxes are calculated by calculate_checkout_line_total and gathered inside base_calculated_subtotal. No need for overloading this method.